content/en/docs/outputs/formatting.md

```diff
@@ -7,48 +7,96 @@ aliases:
 
-The information from k8s/containers is used in conjunction with the command-line options in these ways:
 
-* In rule outputs, if the format string contains `%container.info`, that is replaced with the value from `-pk`/`-pc`, if one of those options was provided. If no option was provided, `%container.info` is replaced with a generic `%container.name (id=%container.id)` instead.
+```yaml
+# Original rule output
+output: Executing binary not part of base image (proc_sname=%proc.sname user=%user.name process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty %container.info)
 
-* If the format string does not contain `%container.info`, and one of `-pk`/`-pc` was provided, that is added to the end of the formatting string.
+# Remove output fields
+output: %proc.sname %user.name
 
-* If `-p` was specified with a general value (i.e. not `-pk`/`-pc`), the value is simply added to the end and any `%container.info` is replaced with the generic value.
+# Add more output fields again
+output: %proc.sname %user.name %proc.is_exe_upper_layer
+```
+
+However, if you want to add the same output field to every rule, doing so manually is tedious. You might seek a shortcut to generically add specific output fields to every rule. The following explanation details how to achieve this.
+
+Falco inherently supports event decoration with associated [Container](https://falco.org/docs/reference/rules/supported-fields/#field-class-container) and [Kubernetes Fields](https://falco.org/docs/reference/rules/supported-fields/#field-class-k8s). To accomplish this, you'll need a special placeholder field in your rules' output fields, denoted as `%container.info`, and you'll need to run Falco with either the `-pk` or `-pc` command-line option. If your rule does not include `%container.info` no container or Kubernetes fields will be added.
+
+Do you have an even more customized use case? We have you covered! We also provide a `-p` flag where you can define custom additional output fields to be included in each rule.
+
+## Example Rule
 
+```yaml
+- rule: Drop and execute new binary in container
+  desc: >
+    SKIPPED
+  condition: >
+    spawned_process
+    and container
+    and proc.is_exe_upper_layer=true 
+    and not container.image.repository in (known_drop_and_execute_containers)
+  output: Executing binary not part of base image (proc_sname=%proc.sname user=%user.name process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty %container.info)
+  priority: CRITICAL
+  tags: [maturity_stable, container, process, mitre_persistence, TA0003, PCI_DSS_11.5.1]
+```
 
-## Examples
+### Scenario 1
 
-Here are some examples of Falco command lines, output strings in rules, and the resulting output:
+The rule outputs include `%container.info`, no command-line flags:
```

Comment:
I don't know what we're saying here with the ", no command-line" flag - do we mean falco wasn't started with -pk or -pc?

If so, maybe it sounds better to say "The rule outputs include `%container.info`,  but Falco was started without the -pk or -pc command-line flags"